api/deploy/kubernetes/cephfs/csidriver.yaml

```diff
@@ -7,3 +7,4 @@ spec:
   attachRequired: false
   podInfoOnMount: false
   fsGroupPolicy: File
+  seLinuxMount: true
```

Comment:
A modification to this file should result in updated generated files under `deploy/` too.

Reply:
`seLinuxMount: true` was added under deploy/ files here - https://github.com/ceph/ceph-csi/commit/c0201e493b7052cb1ec6ef2cecffd8e4cdb49157

Reply:
oh, interesting! In that case we need to run `make deploy` in the CI to check that all modified files are included in a PR. When you run `make deploy`, the `seLinuxMount: true` option should be removed from the `deploy/` directory without your change.

Reply:
well, sortof. See this:
```
$ make -C tools generate-deploy 
make: Entering directory '/var/tmp/ceph-csi/tools'
go run yamlgen/main.go
creating "../deploy/scc.yaml"...done!
creating "../deploy/cephfs/kubernetes/csidriver.yaml"...done!
creating "../deploy/cephfs/kubernetes/csi-config-map.yaml"...done!
creating "../deploy/nfs/kubernetes/csidriver.yaml"...done!
creating "../deploy/nfs/kubernetes/csi-config-map.yaml"...done!
creating "../deploy/rbd/kubernetes/csidriver.yaml"...done!
creating "../deploy/rbd/kubernetes/csi-config-map.yaml"...done!
make: Leaving directory '/var/tmp/ceph-csi/tools'
$ git status
HEAD detached at origin/devel
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   deploy/cephfs/kubernetes/csidriver.yaml
	modified:   deploy/nfs/kubernetes/csidriver.yaml
	modified:   deploy/rbd/kubernetes/csidriver.yaml

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
oh, it seems that this is not checked in Ceph-CSI at all now. We need a workflow similar to kubernetes-csi-addons that does this check:

https://github.com/csi-addons/kubernetes-csi-addons/blob/397940a2db2ed4889e10dcaa2be390e6d97e6db3/.github/workflows/test-golang.yaml#L69-L70

Reply:
created Issue - https://github.com/ceph/ceph-csi/issues/4285